README.md

```diff
@@ -61,7 +61,7 @@ If you are a third party seeking access to VA Lighthouse APIs, you will need to
 
 ## Further Assistance
 
-If you have any questions or need further assistance, please visit our [support page](https://developer.va.gov/support), where you can contact us and peruse the FAQs. You will receive a human reply within 48 hours. Additionally, you may open a comment or support request [inside this repo](https://github.com/department-of-veterans-affairs/vets-api-clients/issues/new/choose).
+If you have any questions or need further assistance, please visit our [support page](https://developer.va.gov/support), where you can contact us and peruse the FAQs. You will receive a human reply within 48 hours.
```

Comment:
There's a mismatch on the estimated time for response. On the dev portal it lists one business day as the expectation.

Reply:
@ryanlink thoughts on this?

Reply:
LGTM!

Reply:
hm.  will take this up with marmoset, hang tight

Reply:
Make it 24 hours to be consistent with the contact us page, please.

Reply:
updated! thanks for clarifying, Ryan, and thanks for catching it Giles